controller.py

```diff
@@ -311,7 +311,9 @@ def updates_feed(self, collection_details):
         precomposed_entries = []
         # Add entries for Works associated with the collection's catalog.
         updated_works = collection.works_updated_since(self._db, last_update_time)
-        works = pagination.apply(updated_works).all()
+        works = pagination.modify_database_query(
+            self._db, updated_works
+        ).all()
```

Comment:
This reflects a change made in core during the Elasticsearch work. `Pagination` can now  modify either a database query or an Elasticsearch query, which made `apply` too generic a name.